src/abstractMatrix.js

```diff
@@ -1506,6 +1508,30 @@ export default function AbstractMatrix(superCtor) {
             return new MatrixSelectionView(this, rowIndices, columnIndices);
         }
 
+        /**
+         * Returns a view of the row indices
+         * @example
+         * // resulting vector is [[1,2,3], [1,2,3]]
+         * var matrix = new Matrix([[1,2,3], [4,5,6]]).selectionRowView([0, 0])
+         * @param {Array<number>} rowIndices
+         * @return {MatrixSelectionRowView}
+         */
+        selectionRowView(rowIndices) {
```

Comment:
I hesitate between that one and also `selectionRowsView`, so the tie breaker @targos